src/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import { email, validatePassCoincidence } from 'utils/inputValidations';
+
+import wLogo from '../../assets/wLogo.png';
+
+import 'scss/layout.scss';
+import 'scss/components.scss';
+import 'scss/margins.scss';
+import styles from './styles.module.scss';
+
+interface Inputs {
```

Comment:
This would be better inside a types.ts file

Reply:
Also rename it i suggest User or some similar